packages/devextreme-schematics/src/add-layout/files/devextreme.json

```diff
@@ -1,35 +1,34 @@
 {
-    "applicationEngine": "<%= engine %>",
-    "build": {
-        "commands": [
-            {
-                "command": "build-theme",
-                "options": {
-                    "inputFile": "<%= sourcePath %>/themes/metadata.base.json",
-                    "outputFile": "<%= sourcePath %>/themes/generated/theme.base.css"
-                }
-            },
-            {
-                "command": "build-theme",
-                "options": {
-                    "inputFile": "<%= sourcePath %>/themes/metadata.additional.json",
-                    "outputFile": "<%= sourcePath %>/themes/generated/theme.additional.css"
-                }
-            },
-            {
-                "command": "export-theme-vars",
-                "options": {
-                    "inputFile": "<%= sourcePath %>/themes/metadata.base.json",
-                    "outputFile": "<%= sourcePath %>/themes/generated/variables.base.scss"
-                }
-            },
-            {
-                "command": "export-theme-vars",
-                "options": {
-                    "inputFile": "<%= sourcePath %>/themes/metadata.additional.json",
-                    "outputFile": "<%= sourcePath %>/themes/generated/variables.additional.scss"
-                }
-            }
-        ]
-    }
+  "applicationEngine": "<%= engine %>",
+  "build": {
+    "commands": [
+      {
+        "command": "build-theme",
+        "options": {
+          "inputFile": "<%= sourcePath %>/themes/metadata.base.json",
+          "outputFile": "<%= sourcePath %>/themes/generated/theme.base.css"
+        }
+      },      {
+        "command": "build-theme",
+        "options": {
+          "inputFile": "<%= sourcePath %>/themes/metadata.additional.json",
+          "outputFile": "<%= sourcePath %>/themes/generated/theme.additional.css"
+        }
+      },
+      {
+        "command": "export-theme-vars",
+        "options": {
+          "inputFile": "<%= sourcePath %>/themes/metadata.base.json",
+          "outputFile": "<%= sourcePath %>/themes/generated/variables.base.scss"
+        }
+      },
+      {
+        "command": "export-theme-vars",
+        "options": {
+          "inputFile": "<%= sourcePath %>/themes/metadata.additional.json",
+          "outputFile": "<%= sourcePath %>/themes/generated/variables.additional.scss"
+        }
+      }
+    ]
+  }
```

Comment:
It looks like this method should be changed accordingly:
https://github.com/DevExpress/devextreme-cli/blob/master/packages/devextreme-schematics/src/add-layout/index.ts#L303
And it's not good that tests does not show that `modifyConfig` can change spaces. I think we should cover it by test.